### PR TITLE
Added 3D-Popup-Cards

### LINKED ISSUE #282: [Feature] Adding 3D Popup Card
### Description

Hi
I want to add a set of 3D Popup Cards in which on hovering the cards, an image will popup a bit.

For better understanding, please see the attached video.

Please assign me this under IWOC2024.

https://github.com/Rakesh9100/Beautiify/assets/123951068/aaf0f1c7-44b6-4b8c-84f4-84e248466fd1

### Screenshots

_No response_

### Checklist

- [X] I have checked the existing issues.
- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/Beautiify/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.
- [X] I am a IWOC'24 contributor.
- [ ] I am a JWOC'24 contributor.
- [ ] I am a SWOC'24 contributor.

### Code of Conduct

- [X] I agree to follow this project's "[Code of Conduct](https://github.com/Rakesh9100/Beautiify/blob/main/.github/CODE_OF_CONDUCT.md)".

### PR DESCRIPTION
**Pull request by learnersurajsingh**
# Fixes Issue🛠️

Closes #282

# Description👨‍💻 
I have added the 3D- Popup Cards using HTML and CSS.

# Type of change📄

- [x] New feature (non-breaking change which adds functionality)

# How this has been tested✅
It has been tested by myself and no bugs are found. Please see the attached video.

# Checklist✅ 

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added demonstration in the form of GIF/video file
- [x] I am an Open Source Contributor

# Screenshots/GIF📷

https://github.com/Rakesh9100/Beautiify/assets/123951068/a59c279d-69fa-4492-930d-eb3dde03d99e

### CODE REVIEW

File: Components/Cards/3D-Popup-Card/index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
...xsvilNM6QnfbMoZBQComhPOyHYZvFtTTYGd/OEGNDi6tzmNnn3Gb+P2hwPoelPDJyDwQpMTBFepwWJ9VLrEYoeVJI+Q2515QECVhuqKuw01kV6nLqkDQeE77xl6bivhyBjjsctkey9Y+xn56qXw3WBSZEGjcJ2iMF1AKed0h526CgtkOQoxTG/hP5A8nP8ASPBTPtzG7z7DoP3hXk/7TsmnYKvdfBf/AMoxPHZURyFSDT4Wh3gppc1wokFOZM7c7oRvutJX+PJ4RjeOyAcDwsJLrgae/GUjwyNxUzy+Rzj3T71hBw07hCwQbWAxQlZpJ+YLisvWTtHmc4BbwgAL2UkjWNslTTukJ8eMhzkcuMmAl7UC4NGy1OLxsdlqd4VO8hU7ytGxOopjfdaP/sVo/wDsU1rtR3Wl226p/lU/yiX6xsFqf4XxD3aUXN0nsmPbpG6DS4WM9Lf+V8Nn/KDWjsqC0N5rK1+8iB4WDcA0tpBY8n4JA5KLHbbKUU4bIEEbhAEmlFI6KQFpWFxDJowb3rcZesEHQO4GVFVlSg/kCllbGCSd/ClkdI672yrY7UhkeiAXI385M3cT0PNBA0U07ZM5cvGYovP4zcy27CysJ6fqAc9qbBE0ABg+p2ULy14KBBCxD9UhHYL9LFinN25QiYWN23pOgd2RicOyw+IfC6wCsLjIpxzR8FeqHVLQdwEAUSrzEnw91JIZCSTkDwU5xKGYzwouTJnJ/PRI7KN2TP7fldhmzcn80qIPlRYeWU7DZQYNkY33KA9kWnyr8/VinqMgncIkuJKvlY3lm6j/AI2/jPS3wg0CqFUnsDzZKELE7Dnsv8dwRieOyf8AJynOu0M+bQyHRgh8xPshwmd/zmTQtE2rTDRyj/t+V4ycaChhe+g1p33UGAqi8prGt4GV5WN/qnPGcsUf8bfx17+UPBT3BrSSpZC95XlDJpAcDVqxqJI2Vi9kUOjB0GkrW2kxx3oXutTvC1u8J7zxWYq00ilH/avKoqPDySGgFF6c0feb9kyJjAA1tUr/APhhFY37mhR/Y38de/lOIAs9lPMXuO9AI75D6WEb/puu60iuEwbn85BSc89DGgjcqOPU8tBItYXBNY0a90A1vDayvpsf/Bxf8rEz7R9DGPcG7K7IR2Q4XfoPdDoB3WH/AIWrt+0zk/nIcJ/PRGVhP5x+UO346yiv/9k="
```
Comment (Rakesh9100) [inline Components/Cards/3D-Popup-Card/index.html:16]: Why to use this long data for an image? Pls use some links or include image assets @learnersurajsingh

Comment (learnersurajsingh) [reply Components/Cards/3D-Popup-Card/index.html:16]: Ok give me some time.

Comment (Rakesh9100) [reply Components/Cards/3D-Popup-Card/index.html:16]: Are you working @learnersurajsingh

Comment (learnersurajsingh) [reply Components/Cards/3D-Popup-Card/index.html:16]: Yes sir.

I'll resolve it in 2 hours

Comment (Rakesh9100) [reply Components/Cards/3D-Popup-Card/index.html:16]: Should I close the PR if your 2 hours is done @learnersurajsingh

**Pull request closed without merging**